pinch.cabal

```diff
@@ -70,6 +70,7 @@ library
         Pinch.Internal.Value
         Pinch.Protocol
         Pinch.Protocol.Binary
+        Pinch.Protocol.Compact
```

Comment:
Hey. This project makes use of [hpack](https://github.com/sol/hpack). You should add this entry to package.yaml instead.